src/components/Courses/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react';
+import Input from '../../../../common/Input/Input';
+
+export default function SearchBar({ course, setCourse }) {
+	const searchInfo = 'Search Courses..';
+	const [input, setInput] = useState({});
+
+	const handleChange = (event) => {
+		event.preventDefault();
+		const name = event.target.name;
```

Comment:
Do you really need to store the name of your input field in state? I think it would be enough just to store value